### PR TITLE
(APIS-1072) - FE: Create Storefront base project

### PR DESCRIPTION
**Pull request by pedroCloudoki**
Beginnings of the Storefront's UI boilerplate. Very rudimentary as of yet. Has yet to have a store akin to that of API Suite's Core, as well as translation integration, among other things. Only shows an illustrated error screen for now.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```
Comment (a31859) [inline .env]: Shouldn't the env vars be similar to core?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
+{
+  "name": "storefront-boilerplate",
+  "version": "0.1.0",
+  "author": "Cloudoki",
```
Comment (a31859) [inline package.json]: Change author to APISuite

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (a31859) [inline public/index.html]: content value needs to be updated

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (a31859) [inline public/index.html]: Title needs to be updated

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--
```
Comment (a31859) [inline public/index.html]: Remove unneeded comments

File: public/manifest.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (a31859) [inline public/manifest.json]: This needs to be updated

**Review by a31859 — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (pedroCloudoki) [reply .env]: The `SKIP_PREFLIGHT_CHECK=true` is necessary to let Create React App (what I've used to kickstart this boilerplate) know that it's okay that we're using a different eslint version than the one it initially installs.

As for the remaining environment variables, I've now added them in.

**Review by a31859 — APPROVED** [review]

**Pull request merged**